docs/setup_guides/hpc.rst

```diff
@@ -41,7 +41,9 @@ Create a new conda environment for our pangeo work:
 ::
 
     conda create -n pangeo -c conda-forge \
-        python=3.6 dask distributed xarray jupyterlab mpi4py dask-jobqueue
+        python=3.6 xarray \
+        jupyterlab nbserverproxy \
+        dask distributed mpi4py dask-jobqueue
```

Comment:
I take it that we're still using the `dask-mpi` solution here?

Reply:
It is mentioned as an option at the bottom of the page.